lnrpc/peersrpc/peers.proto

```diff
@@ -1,10 +1,90 @@
 syntax = "proto3";
 
+import "lightning.proto";
+
 package peersrpc;
 
 option go_package = "github.com/lightningnetwork/lnd/lnrpc/peersrpc";
 
 // Peers is a service that can be used to get information and interact
 // with the other nodes of the newtwork.
 service Peers {
+    /*
+    UpdateNodeAnnouncement allows the caller to update the node parameters
+    and broadcasts a new version of the node announcement to its peers.
+    */
+    rpc UpdateNodeAnnouncement (NodeAnnouncementUpdateRequest)
+        returns (NodeAnnouncementUpdateResponse);
+}
+
+enum UpdateAction {
+    ADD = 0;
```

Comment:
Should add a comment for the API docs (even if the name is very self-explanatory).

Reply:
Forgot to push this change too?